<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>8</width>
        <height>8</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <widget class="QWidget" name="polarisation_ellipse_tab">
       <attribute name="title">
        <string>Polarisation Ellipse</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="2">
         <widget class="QGroupBox" name="polarisation_ellipse_information">
          <property name="title">
           <string>Information</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0" colspan="2">
            <widget class="QPushButton" name="rand_stokes_button_xyplot">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Generate Random Polarisation State</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="s0_label_xyplot">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>S0:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="s0_value_xyplot">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0.000</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="s1_label_xyplot">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>S1:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="s1_value_xyplot">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0.000</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="s2_label_xyplot">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>S2:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="s2_value_xyplot">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0.000</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="s3_label_xyplot">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>S3:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="s3_value_xyplot">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0.000</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="dop_label_xyplot">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>DOP:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="dop_value_xyplot">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0.00%</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="PlotWidget" name="polarisation_ellipse_widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>451</width>
            <height>451</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>451</width>
            <height>451</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="stokes_parameters_tab">
       <attribute name="title">
        <string>Stokes Parameters</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="PlotWidget" name="stokes_parameters_widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>451</width>
            <height>451</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>451</width>
            <height>451</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="stokes_parameters_information">
          <property name="title">
           <string>Information</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0" colspan="2">
            <widget class="QPushButton" name="rand_stokes_button_barchart">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Generate Random Polarisation State</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="s0_label_barchart">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>S0:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="s0_value_barchart">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0.000</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="s1_label_barchart">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>S1:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="s1_value_barchart">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0.000</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="s2_label_barchart">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>S2:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="s2_value_barchart">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0.000</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="s3_label_barchart">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>S3:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="s3_value_barchart">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0.000</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="dop_label_barchart">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>DOP:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="dop_value_barchart">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0.00%</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="poincare_sphere">
       <attribute name="title">
        <string>Poincar√© Sphere</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <widget class="QGroupBox" name="poincare_sphere_information">
          <property name="title">
           <string>Information</string>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0" colspan="2">
            <widget class="QPushButton" name="rand_stokes_button_xyzplot">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Generate Random Polarisation State</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="s0_label_xyzplot">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>S0:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="s0_value_xyzplot">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0.000</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="s1_label_xyzplot">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>S1:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="s1_value_xyzplot">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0.000</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="s2_label_xyzplot">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>S2:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="s2_value_xyzplot">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0.000</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="s3_label_xyzplot">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>S3:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="s3_value_xyzplot">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0.000</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="dop_label_xyzplot">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>DOP:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="dop_value_xyzplot">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0.00%</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="GLViewWidget" name="poincare_sphere_opengl_widget">
          <property name="minimumSize">
           <size>
            <width>451</width>
            <height>451</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>451</width>
            <height>451</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="status_tab">
       <attribute name="title">
        <string>Status</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="settings_tab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLViewWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>pyqtgraph.opengl</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
